Dockerfile

```diff
@@ -11,7 +11,7 @@ FROM krallin/ubuntu-tini as ubuntu-tini
 
 # rethinkdb backup/restore requires the python client library
 # let's make small binaries of these commands in order not to blow up the image size
-FROM rethinkdb:2.4.0 as rethinkdb-python-client-builder
+FROM rethinkdb:2.4.1 as rethinkdb-python-client-builder
```

Comment:
I don't know if this should really be necessary. It does not resolve our original problem with the latest releases rethinkdb made.

Reply:
Done